ui/narrative/methods/import_attribute_mapping_from_staging/display.yaml

```diff
@@ -21,16 +21,14 @@ suggestions :
 
 parameters :
     staging_file_subdir_path :
-        ui-name : |
-            File path
+        ui-name : File path
         short-hint : |
             File with tabular Attribute Mapping data.
         long-hint  : |
             Recognized extensions for the input file are .tsv or .xlsx
 
     attribute_mapping_name :
-        ui-name : |
-            Attribute Mapping object name
+        ui-name : Attribute Mapping object name
```

Comment:
Not sure if mapping should be capitalized or not

Reply:
There seemed to be capitalisation of KBase types, so I left that as-is; similarly 'Narrative' is capitalised all over the place, which is obviously _deeply_ wrong, but I let that go.